### PR TITLE
Translate 2018-03-00-How-To-Integrate-Rollingupdate-Strategy.md

### PR DESCRIPTION
**Pull request by yixin21**
zh-trans 2018-03-00-How-To-Integrate-Rollingupdate-Strategy.md

ref:k8smeetup/website-tasks#3473

### CODE REVIEW

**Review by markthink — COMMENTED** [review]
/lgtm
/approve

File: content/zh/blog/_posts/2018-03-00-How-To-Integrate-Rollingupdate-Strategy.md [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+<!--
+###  Controller revision in WQ-RDS
+
+Revision information is stored in MysqlCluster.Status and is no different than Statefulset.Status.
+-->
+###  WQ-RDS 中的控制器修改
+
+修订信息存储在 MysqlCluster.Status 和 Statefulset.Status 中的没有差别。
+
+```
```
Comment (tengqm) [inline content/zh/blog/_posts/2018-03-00-How-To-Integrate-Rollingupdate-Strategy.md:248]: The indentation of the sample is bad ... pls fix.

File: content/zh/blog/_posts/2018-03-00-How-To-Integrate-Rollingupdate-Strategy.md [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+![][11]
+
+![][12]
+
+![][13]
+
+![][14]
+
+![][15] 
+
```
Comment (tengqm) [inline content/zh/blog/_posts/2018-03-00-How-To-Integrate-Rollingupdate-Strategy.md:345]: why we are getting so many empty lines?

**Comment by tengqm** [comment]
Temporarily closing this because there have been no action for over a month. There are errors in the markdown to be fixed for this to be merged.  Please reopen when you are ready to fix and commit again. Thanks.
/close

**Pull request closed without merging**